static/app/views/eventsV2/results.tsx

```diff
@@ -81,10 +81,16 @@ function readShowTagsState() {
   return value === '1';
 }
 
-function getYAxis(location: Location, savedQuery?: SavedQuery) {
-  return location.query.yAxis
-    ? decodeList(location.query.yAxis)
-    : decodeList(savedQuery?.yAxis);
+function getYAxis(location: Location, eventView: EventView, savedQuery?: SavedQuery) {
+  if (location.query.yAxis) {
+    return decodeList(location.query.yAxis);
+  }
+  if (location.query.yAxis === null) {
+    return [];
+  }
+  return savedQuery?.yAxis && savedQuery?.yAxis.length > 0
+    ? decodeList(savedQuery?.yAxis)
+    : [eventView.getYAxis()];
```

Comment:
If the user clears all the values by interacting with the dropdown menu, `location.query.yAxis` will be `null`. In that case, we return an empty array without adding any default value to it. 
On the other hand, if the user opens a URL with no `yAxis` parameter in the query, then `location.query.yAxis` will be `undefined`, and we proceed to use either the saved query or default value (`[eventView.getYAxis()]`).